src/app/domain/Part.ts

```diff
@@ -7,10 +7,12 @@ import { GameInfo } from '../components/normal-component/pick-game/pick-game.com
 export type Part = {
     readonly typeGame: string; // the type of game
     readonly playerZero: MinimalUser; // the first player
+    readonly playerZeroElo: number, // the elo of the first player at the beginning of the part
     readonly turn: number; // -1 means the part has not started, 0 is the initial turn
     readonly result: IMGPResult;
 
     readonly playerOne?: MinimalUser; // the second player
+    readonly playerOneElo?: number, // must be modified/set whenever playerOne is modified/set
```

Comment:
le commentaire décrit pas ce que c'est, à changer par // the elo of the second player
Peut-être juste ajouter un commentaire avant le groupe de `?` qui dit: The following fields are set when relevant

Reply:
il est supprimé entièrement ce champ donc ? Si oui, TPR